### ISSUE TITLE
Add e2e tests for air gap installer feature

### ISSUE THREAD

**Issue by mukundansundar** [dapr/cli#928]
## Describe the proposal

Add e2e tests for air gap installer feature

Get any one of the released installer-bundles, use that to test the air gap installer feature in dapr cli e2e. 

## Release Note

RELEASE NOTE:

**Comment by shivamkm07** [comment]
> @shivamkm07 Will you picking up this issue?

Yes the associated PR https://github.com/dapr/cli/pull/1198 needs to be updated, will do so. Thanks for reminding
